doc/api/buffer.md

```diff
@@ -1003,7 +1003,8 @@ changes:
     description: Additional parameters for specifying offsets are supported now.
 -->
 
-* `target` {Buffer|Uint8Array} A `Buffer` or [`Uint8Array`] to compare to.
+* `target` {Buffer|Uint8Array} A `Buffer` or [`Uint8Array`] with which to
```

Comment:
We might want to rename `target` to `otherBuffer` or rename `otherBuffer` to `target` down below. That way it is more consistent.